packages/pigeon/platform_tests/alternate_language_test_plugin/android/src/main/java/com/example/alternate_language_test_plugin/CoreTests.java

```diff
@@ -1269,11 +1271,20 @@ void callFlutterEchoNullableEnum(
      */
     static void setUp(
         @NonNull BinaryMessenger binaryMessenger, @Nullable HostIntegrationCoreApi api) {
+      setUp(binaryMessenger, "", api);
+    }
+
+    static void setUp(
+        @NonNull BinaryMessenger binaryMessenger,
+        @NonNull String messageChannelSuffix,
+        @Nullable HostIntegrationCoreApi api) {
       {
+        messageChannelSuffix = messageChannelSuffix.isEmpty() ? "" : "." + messageChannelSuffix;
```

Comment:
Instead of having this generated at the start of every method handler, could this be generated as the first line of `setUp`. This also applies to the other languages.

Reply:
Yeah, I did that on the others, I just missed this one.